config/locales/biomodelos.es.yml

```diff
@@ -48,7 +48,7 @@ es:
         text_2: 'Puedes aportar a mejorar el conocimiento sobre la distribución geográfica de las especies a través de las siguientes actividades:'
         activities_list_html: >
                                 - Curación de registros biológicos.</br>
-                                - Edición de los modelos estadísticos desarrollados por el Laboratorio de Biogeografía Aplicada.</br>

```

Comment:
I assume you confirmed with Cesar or someone from the BioModelos team right? Since Elkin said PEM in the email

Reply:
Yes, it is confirmed with Cesar